recipes-browser/cog/cog_0.14.1.bb

```diff
@@ -0,0 +1,16 @@
+require cog.inc
+require conf/include/devupstream.inc
+
+SRC_URI[sha256sum] = "fb91104e25e1dde27189c91c70acc356e387f47acebaa8997e01ce5879c3a600"
+
+SRC_URI:class-devupstream = "git://github.com/Igalia/cog.git;protocol=https;branch=cog-0.14"
+SRCREV:class-devupstream = ""dcedb3fb7541e78a808026949c9b83192d5f8a4d
+
+RDEPENDS:${PN} += "wpewebkit (>= 2.36)"
```

Comment:
I had to change this to remove the version... otherwise it kept complaining about missing wpewebkit 2.36.  Not sure if it was due to my other changes, but just in case.

Reply:
```
build@canteira:~/perf-browser-bsp/builds/raspberrypi3-mesa-wpe-nightly$ bitbake-layers show-recipes wpewebkit
NOTE: Starting bitbake server...
WARNING: You have included the meta-security layer, but 'security' has not been enabled in your DISTRO_FEATURES. Some bbappend files and preferred version setting may not take effect. See the meta-security README for details on enabling security support.
Loading cache: 100% |#########################################################################################################################################################| Time: 0:00:00
Loaded 4735 entries from dependency cache.
Parsing recipes: 100% |#######################################################################################################################################################| Time: 0:00:00
Parsing of 3037 .bb files complete (3034 cached, 3 parsed). 4738 targets, 122 skipped, 0 masked, 0 errors.
WARNING: No recipes in default available for:
  /home/build/perf-browser-bsp/sources/meta-perf-browser/recipes-browser/cog/cog_0.10.%.bbappend
  /home/build/perf-browser-bsp/sources/meta-perf-browser/recipes-browser/cog/cog_0.8.%.bbappend
  /home/build/perf-browser-bsp/sources/meta-perf-browser/recipes-bsp/u-boot-fslc/u-boot-fslc_%.bbappend
  /home/build/perf-browser-bsp/sources/meta-perf-browser/recipes-core/images/initramfs-ostree-image.bbappend

Summary: There were 2 WARNING messages shown.
WARNING: preferred version 13.0.0 of nativesdk-llvm not available
WARNING: versions of nativesdk-llvm available: 12.0.1
WARNING: preferred version 13.0.0 of llvm-native not available
WARNING: versions of llvm-native available: 12.0.1
WARNING: preferred version 13.0.0 of llvm not available
WARNING: versions of llvm available: 12.0.1
WARNING: preferred version trunk+% of cog not available
WARNING: versions of cog available: 0.12.4 0.14+httpsAUTOINC+343563b086 0.14.0
```

Reply:
Clearly the `meta-perf-browser` layer is doing some kind of wrong overwrite on the recipes.  I could suggest to you to test the build following the instructions documented on https://github.com/Igalia/meta-webkit/wiki/RPi